webpack.config.js

```diff
@@ -0,0 +1,7 @@
+const defaultScriptsConfig = require( '@wordpress/scripts/config/webpack.config' );
```

Comment:
Is there a way to avoid making this file just for the sake of doing this? I might be missing something obvious.

Reply:
I don't know of an easy way, based off of the changes to `package.json`. This is because we're using `webpack` specifically instead of `wp-scripts`, but we want to use all of the `wp-scripts` defaults. This file is doing that.

I think this is fine, though. If we do want to adjust builds, add HMR, or anything like that, it's nice to have direct access to the script defaults to edit.

Reply:
I think you could simplify the below to:

```
module.exports = [ defaultScriptsConfig ];
```

and maybe leave a comment about why we're using `webpack` directly instead of `wp-scripts build` and family.

Reply:
> I think this is fine, though. If we do want to adjust builds, add HMR, or anything like that, it's nice to have direct access to the script defaults to edit.

Ah, that's why I had done it this way in the first place! I was hoping to see if I could simplify it further but this is good. I'll make that one line change you are suggesting though, better to do that for now.

Reply:
Pushed it up